app/mainscreen/monthlyview/ProgressBar.tsx

```diff
@@ -0,0 +1,36 @@
+"use client";
+import { useState } from "react";
+
+interface IProgressBar {
+  income: number;
+  expense: number;
+  dest: number;
+}
+
+export default function ProgressBar({ expense, income, dest }: IProgressBar) {
+  const [expensesWidth, setExpensesWidth] = useState((expense / income) * 100);
+  const [destPlacement, setDestPlacement] = useState((dest / income) * 100);
+
+  return (
+    <section className="w-full bg-green-primary h-[18px] flex rounded-[10px] relative">
```

Comment:
We should try to avoid using arbitrary numbers like 18px and 10px.
Just use the closest one , i.e: h-4 (16px) or h-5 (20px), and rounded-lg (8px)